src/dhali/transaction_utils.py

```diff
@@ -1,39 +1,39 @@
 import json
 import xrpl
-from fastapi import HTTPException
 import uuid
+from fastapi import HTTPException
 from google.cloud import firestore
 
-
-@firestore.transactional
-def transactional_validation(
+def _transactional_validation(
```

Comment:
I know you mentioned today that you were making it none transactional, but I wasn't expecting to see that you'd completely removed the decorator so that none of our firestore interactions are transactional

Reply:
Whoops, that was a mistake!  Good catch :joy: